app/controllers/git_api/tracks_controller.rb

```diff
@@ -63,5 +65,17 @@ def query
         }
       }
     end
+
+    def set_track
```

Comment:
This is quite important in Rails. because controller's methods can be exposed via the router, having things as `private` guarantees that a user won't be able to call this method directly.

```suggestion
    private
    def set_track
```

Reply:
Doh